developerportal/settings/base.py

```diff
@@ -336,3 +314,55 @@
         "OPTIONS": {"CLIENT_CLASS": "django_redis.client.DefaultClient"},
     }
 }
+
+# Mozilla OpenID Connect / Auth0 configuration
+
+# How frequently do we check with the provider that the
+# user still exists and is authorised?
+OIDC_RP_SIGN_ALGO = "RS256"
+
+OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = (
+    15 * 60
+)  # 15 mins is the default, but re-expressed here for visibility
```

Comment:
Do we need to have active code here at all, or can we just leave a comment along the lines of:

```
# Tokens expire every 15 minutes by default
# To override, set OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS
```